samples/oauth_rails_vethealth/app/models/oauth_callback.rb

```diff
@@ -0,0 +1,47 @@
+class OauthCallback < ApplicationRecord
+  serialize :response_body_raw
+
+  has_one :authentication
+
+  def auth(protect_secrets=true)
```

Comment:
Could you explain when/how this would be false?

Reply:
I set it to false when I want to use the real values in the authentication call.
https://github.com/department-of-veterans-affairs/vets-api-clients/blob/6328c494404351c2d33e4c3afba2211947506f95/samples/oauth_rails_vethealth/app/models/oauth_callback.rb#L26
But when displaying `auth` in a view it will default to hiding the secrets